2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/Chassis.java

```diff
@@ -102,9 +102,9 @@ public void Shifting_points(double up, double down){
     }
 
     @Log
-    private boolean allowPIDChanges;
+    private boolean allowPIDChanges = true;
```

Comment:
We should remove `allowPIDChanges` because it is pretty confusing during troubleshooting. But for now let's merge this.